aspnetcore/signalr/webpack-and-typescript.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Using ASP.NET Core SignalR with TypeScript and WebPack
+author: ssougnez
+description: How to bundle a web application using SignalR, TypeScript and WebPack.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: ssougnez
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/webpack-and-typescript
+---
+
+# Using ASP.NET Core SignalR with TypeScript and WebPack
+
+By [Sébastien Sougnez](https://twitter.com/ssougnez)
+
+**WebPack** enables developers to easily bundle the client side of a web application. This tutorial teaches how to use it to bundle an application written in **TypeScript** that communicates with a server through web sockets using **SignalR**.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/live/aspnetcore/signalr/webpack-and-typescript/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Create the ASP.NET Core web application
+
+# [Visual Studio](#tab/visual-studio/)
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**. Give a meaningful name to the application and validate.
+2. Select **Empty** and ensure that **ASP.NET Core 2.1** is selected from the framework selector.
+
+   ![New Project dialog in Visual Studio](webpack-and-typescript/_static/signalr-create-web-application.png)
+
+# [Visual Studio Code](#tab/visual-studio-code/)
+
+1. From the **Integrated Terminal**, run the following command:
+
+    ```console
+    dotnet new web -o SignalRWebPack
+    ```
+---
+
+## Configure WebPack and TypeScript
+
+To be able to bundle all front end related resources (stylesheets, images and **TypeScript** files), you need to install the following packages using your prefered packages manager.
+
+1. Run the following command to install the required packages for **WebPack**.
+
+    ```console
+    yarn add clean-webpack-plugin css-loader html-webpack-plugin mini-css-extract-plugin ts-loader typescript webpack webpack-cli -D
```

Comment:
I don't think you should use yarn here, npm works better and it's 1 dependency less. 

npm add clean-webpack-plugin css-loader html-webpack-plugin mini-css-extract-plugin ts-loader typescript webpack webpack-cli -D

Reply:
Will do, although from my experience, yarn works way better than npm hehe but it does not really matter, I’ll change that ;-)

Reply:
yarn -v- npm is one of those topics :)